spec/controllers/hyrax/batch_edits_controller_spec.rb

```diff
@@ -127,6 +129,7 @@
         expect { Collection.find(collection1.id) }.not_to raise_error(Ldp::Gone)
         expect { Collection.find(collection2.id) }.not_to raise_error(Ldp::Gone)
         expect { Collection.find(collection3.id) }.to raise_error(Ldp::Gone)
+        expect(flash[:notice]).to eq("One or more of the selected collections could not be deleted. You cannot delete a collection for which you have read-only access. In addition, you cannot delete a collection of the type \"Admin Set\" unless that collection is empty.")
       end
     end
   end
```

Comment:
The tests look good.  I like the coverage of collections.  Can you add one that tests including an empty admin set and an admin set with works?